### DISCUSSION TITLE
Render image media field in Liquid

### DISCUSSION THREAD

**Discussion by Diman96** [OrchardCMS/OrchardCore#9835] [Q&A]
I need to render `alt` and `title` attributes for images in Liquid.
What is the best way to do that? Creating a tag or a filter where the media field will be parsed and converted into an img tag completely or img attributes, depending on what is located there (In media field)?

https://github.com/OrchardCMS/OrchardCore/issues/9710

**Comment by PiemP** [comment]
you could create your `MyImagePart` with and `ImageField` and a `TextField` for the `title` attribute. Add your `MyImagePart` to your content type and create the related shapes to render the `MyImagePart` in every situations.

**Comment by ns8482e** [comment]
@Diman96   See how it's done in Agency theme 

https://github.com/OrchardCMS/OrchardCore/blob/e2a1b00142fa7703de35ed57ab504cf0ff658952/src/OrchardCore.Themes/TheAgencyTheme/Recipes/Snippets/landingpage.liquid#L52

Change `alt` to `title`

You can also create template for Media [field alternates](https://docs.orchardcore.net/en/dev/docs/reference/modules/Templates/index.html#content-field-templates).
